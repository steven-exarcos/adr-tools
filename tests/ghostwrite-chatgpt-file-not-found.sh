adr init
adr new Test Decision
if adr ghostwrite chatgpt 9999
then
    echo ERROR: should have failed
fi