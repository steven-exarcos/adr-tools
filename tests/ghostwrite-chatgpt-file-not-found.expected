adr init
doc/adr/0001-record-architecture-decisions.md
adr new Test Decision
doc/adr/0002-test-decision.md
if adr ghostwrite chatgpt 9999
then
    echo ERROR: should have failed
fiError: No ADR found with number 9999
